test/unit/test_introspect.py

```diff
@@ -83,3 +121,252 @@ def test_yaml_extension(data_dir):
         'python': {'test_collection.test_yaml_extension': ['python-six']},
         'system': {},
     }
+
+
+def test_sanitize_pep508():
```

Comment:
```suggestion
def test_filter_requirements():
```

Let us just drop `sanitize` word.

Reply:
Updated in latest commit. Good catch.